src/main/scala/org/allenai/deep_qa/data/Instance.scala

```diff
@@ -108,15 +108,14 @@ case class SnliInstance(
   */
 case class WhoDidWhatInstance(
```

Comment:
This is now identical to the `McReadingComprehensionInstance`, so you can probably just remove it.